src/server/scripts/Spells/spell_monk.cpp

```diff
@@ -432,6 +436,42 @@ class spell_monk_save_them_all : public AuraScript
     }
 };
 
+// 198898 - Song of Chi-Ji
+struct at_monk_song_of_chi_ji : AreaTriggerAI
+{
+    using AreaTriggerAI::AreaTriggerAI;
+
+    void OnInitialize() override
+    {
+        SpellInfo const* spellInfo = sSpellMgr->GetSpellInfo(at->GetSpellId(), DIFFICULTY_NONE);
+        if (!spellInfo)
+            return;
+
+        Unit* caster = at->GetCaster();
+        if (!caster)
+            return;
+
+        Position destPos = at->GetFirstCollisionPosition(40.0f, 0.0f);
```

Comment:
hardcoded distance, use the spell's max distance instead, which is how far it travels